NewInternalsSpecification.html

```diff
@@ -0,0 +1,271 @@
+    <emu-alg>
+      1. Let _doesAccept_ be *false*.
+      1. For each _accept_ in _acceptList_, do
+        1. Let _activeChangeCount_ be Get(_activeChanges_, _accept_).
+        1. ReturnIfAbrupt(_activeChangeCount_).
+        1. If _activeChangeCount_ > 0, return *false*.
+        1. If _accept_ is same string as _changeType_, then
+          1. Let _doesAccept_ be *true*  .
+      1. return _doesAccept_.
+    </emu-alg>
+  </emu-clause>
+
+  <emu-clause id="EnqueueChangeRecord" aoid="EnqueueChangeRecord">
+    <h1>EnqueueChangeRecord(O, changeRecord)</h1>
+
+    <p>When the abstract operation EnqueueChangeRecord is called with Object _O_ and change record _changeRecord_ the following steps are taken:</p>
+    <emu-alg>
+      1. Assert: Type(_O_) is Object.
+      1. Let _notifier_ be GetNotifier(_O_).
+      1. Let _changeType_ be  Get(_changeRecord_, `"type"`).
+      1. ReturnIfAbrupt(_changeType_).
+      1. Let _activeChanges_ be the value of _notfier_'s [[ActiveChanges]] internal slot.
+      1. Let _changeObservers_ be the value of _notifier_'s [[ChangeObservers]] internal slot.
+      1. For each _observerRecord_ in _changeObservers_, do
+        1. Let _acceptList_ be Get(_observerRecord_, `"accept"`).
+        1. ReturnIfAbrupt(_acceptList_).
+        1. Let _deliver_ be ShouldDeliverToObserver(_activeChanges_, _acceptList_, _changeType_).
+        1. If _deliver_ is *false*, continue.
+        1. Otherwise, let _observer_ be Get(_observerRecord_, `"callback"`).
+        1. ReturnIfAbrupt(_observer_).
+        1. Let _pendingRecords_ be the value of _observer_'s [[PendingChangeRecords]] internal slot.
+        1. Append _changeRecord_ to the end of _pendingRecords_.
+    </emu-alg>
+  </emu-clause>
+
+
+  <emu-clause id="DeliverChangeRecords" aoid="DeliverChangeRecords">
+    <h1>DeliverChangeRecords(C)</h1>
+
+    <p>When the abstract operation DeliverChangeRecords is called with callback _C_, the following steps are taken:</p>
+
+    <emu-alg>
+      1. Let _changeRecords_ be the value of _C_'s [[PendingChangeRecords]] internal slot.
+      1. Set _C_'s [[PendingChangeRecords]] internal slot to a new empty List.
+      1. Let _array_ be ArrayCreate(0).
+      1. Let _n_ be 0.
+      1. For each _record_ in _changeRecords_, do:
+        1. Let _status_ be the result of CreateDataProperty(_array_, ToString(_n_), _record).
+        1. Assert: _status_ is *true*.
+        1. Increment _n_ by 1.
+      1. If _array_ is empty, return *false*.
+      1. Let _status_ be Call(_C_, *undefined*, «_array_»).
+      1. _result_ is intentionall ignored here since. An implementation might choose to log abrubt completions here.
+      1. Return *true*.
+    </emu-alg>
+
+    <emu-note>The user facing function ''Object.deliverChangeRecords'' returns *''undefined''* to prevent detection if anything was delivered or not.</emu-note>
```

Comment:
Agreed, those were in the source document, not really sure what they're for. Looks like I can just replace with single back-ticks for the first and just * for the second.